### DISCUSSION TITLE
I'm having internet issues after use peacock

### DISCUSSION THREAD

**Discussion by namimama** [thepeacockproject/Peacock#386] [Q&A] [answered]
as a mencioned i'm have some issues, i play the game with peacock but when i close the game and close peacock launcher and Start Server, if i go play some league of legends in my champ select,my screen gets stuck, I can't pick up any champions and then a connection warning pops up, it happened once I thought it was my net but it happened again today.

**Comment by RDIL** [comment, marked as answer]
Assuming everything related to Peacock is closed, this can’t be a Peacock issue. The server won’t occupy the port after its closed, and the patcher only touches the game, not other processes.

**Reply by namimama** [reply]
so it's probably something with my internet, thanks for clarifying
